### PR TITLE
Docs: initial documentation for linelist

### PR DESCRIPTION
**Pull request by ntejos**
+ other minor tweak at linelist itself.

### CODE REVIEW

File: docs/linelist.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Property     Variable  Type        Description
+============ ========= =========== ====================================================
+LineList key llst_keys str or list A key to define a subsample of transitions to load:
+                                     'ISM' - "All" ISM lines (can be overwhelming!)
+                                     'Strong' - Strong ISM lines
+                                     'HI' - HI Lyman series
+                                     'H2' - H2 (Lyman-Werner)
+                                     'CO' - CO UV band-heads
+                                     'EUV' - Extreme UV lines
+============ ========= =========== =====================================================
```
Comment (nhmc) [inline docs/linelist.rst]: I think the table and the sentence above it is confusing. How about a list of bullet points, for 'ISM', 'strong', etc?

File: docs/linelist.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+   /home/ntejos/python/linetools/linetools/data/lines/verner94_tab6.fits
+  linetools.lists.parse: Reading linelist --- 
+   /home/ntejos/python/linetools/linetools/data/lines/EUV_lines.ascii
+  read_sets: Using set file -- 
+  /home/ntejos/python/linetools/linetools/lists/sets/llist_v0.4.ascii
+
+In these examples, the object ``hi`` has purely HI Lyman series
+transitions (e.g. HI Lya) and ``euv`` has HI Lyman series and Extreme
+UV transitions (e.g. Ne VIII, MgX). The available keys are listed in
+Table XX.
```
Comment (nhmc) [inline docs/linelist.rst]: Need to add a reference label here.

**Comment by nhmc** [comment]
I left some inline comments. I looks good overall, thanks!

Have you run a sphinx build to check that there are no warnings? If not, please do that. Use 
```
python setup.py build_doc
```
You'll also have to add a reference to linelist.rst in the index.rst file, so it shows up in the docs.

(I have to update the doc build instructions).

File: docs/linelist.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+* 'ISM' : "All" ISM lines (can be overwhelming!)
+* 'Strong' : Strong ISM lines (most common absorption line transitions observed)
+* 'HI' : HI Lyman series
+* 'H2' : H2 (Lyman-Werner)
+* 'CO' : CO UV band-heads
+* 'EUV' :  Extreme UV lines
+
+We note that a subset of transitions can also be specified using the
+`subset_line()` methods (see Section XXX). In this way, one can create
```
Comment (nhmc) [inline docs/linelist.rst]: Need to add a reference here. If you want to add a placeholder for future docs covering subset_line, please put it in a comment (see Absline Class comment above).

**Comment by nhmc** [comment]
Looks good. After addressing the inline comment, please merge.

**Comment by ntejos** [comment]
Will merge now; the documentation in linelist is still incomplete so I will continue with it in the near future... hopefully addressing the label thing in a better way than currently.

**Pull request merged**